### PR TITLE
Add get running task api

### PR DESCRIPTION
**Pull request by An-Li-magicloud**

### CODE REVIEW

File: tidy3d/web/simulation_task.py [hunk 1]
```diff
@@ -228,6 +228,29 @@ def get(cls, task_id: str) -> SimulationTask:
...
+    @classmethod
+    def get_running_tasks(cls) -> [SimulationTask]:
+        """Get running tasks from the server.
+
+        Parameters
+        ----------
+
+        Returns
+        -------
+        :class:`[.SimulationTask]`
```
Comment (tylerflex) [inline tidy3d/web/simulation_task.py:240]: let's do
```
List[:class:`.SimulationTask`]
```
or else it won't render properly in our docs.

File: tidy3d/web/simulation_task.py [hunk 2]
```diff
@@ -228,6 +228,29 @@ def get(cls, task_id: str) -> SimulationTask:
         resp = http.get(f"tidy3d/tasks/{task_id}/detail")
         return SimulationTask(**resp) if resp else None
 
+    @classmethod
+    def get_running_tasks(cls) -> [SimulationTask]:
```
Comment (tylerflex) [inline tidy3d/web/simulation_task.py:232]: our convention is to use
```py
List[SimulationTask]
```
with
```py
from typing import List
```

File: tidy3d/web/simulation_task.py [hunk 3]
```diff
@@ -228,6 +228,29 @@ def get(cls, task_id: str) -> SimulationTask:
...
+        Parameters
+        ----------
+
+        Returns
+        -------
+        :class:`[.SimulationTask]`
+            :class:`.SimulationTask` object containing info about status,
+             size, credits of task and others.
+        """
+        resp = http.get("tidy3d/py/tasks")
```
Comment (tylerflex) [inline tidy3d/web/simulation_task.py:244]: can we split this up a little to make it easier to read?
```py
if not resp:
    return []
return parse_obj_as(List[SimulationTask], resp)
```

File: tidy3d/web/simulation_task.py [hunk 4]
```diff
@@ -228,6 +228,29 @@ def get(cls, task_id: str) -> SimulationTask:
         resp = http.get(f"tidy3d/tasks/{task_id}/detail")
         return SimulationTask(**resp) if resp else None
 
+    @classmethod
+    def get_running_tasks(cls) -> [SimulationTask]:
+        """Get running tasks from the server.
```
Comment (tylerflex) [inline tidy3d/web/simulation_task.py:233]:
```
"Get a list of running tasks from the server"
```

just to be a little more specific that it's a list

**Review by tylerflex — CHANGES_REQUESTED** [review]
Could you add a line in CHANGELOG.md under "Added"
```
- Add `SimulationTask.get_running_tasks()` to get a list of running tasks from the server.
```

@An-Li-magicloud is this finished or do you have more changes to add?

**Comment by An-Li-magicloud** [comment]
> Could you add a line in CHANGELOG.md under "Added"
> 
> ```
> - Add `SimulationTask.get_running_tasks()` to get a list of running tasks from the server.
> ```
> 
> @An-Li-magicloud is this finished or do you have more changes to add?

This is part of https://flow360.atlassian.net/browse/SCEM-2452 . 

It could help user to get running tasks if connection issue or python kernel gone.

Let's merge this and discuss more about how to improve monitor in the ticket.

**Review by tylerflex — APPROVED** [review]

**Pull request merged**